src/contracts/core/StrategyManager.sol

```diff
@@ -28,6 +30,25 @@ contract StrategyManager is
 {
     using SafeERC20 for IERC20;
 
+    function stakerStrategyShares(address staker, IStrategy strategy) public view returns (uint256) {
+        address operator = delegation.delegatedTo(staker);
+        uint64 scalingFactor = slasher.shareScalingFactor(operator, strategy);
+        return SlashingAccountingUtils.normalize({
+            nonNormalizedShares: nonNormalizedStakerStrategyShares[staker][strategy],
+            scalingFactor: scalingFactor
+        });
+    }
+
+    // includes the effect of all unexecuted but pending slashings
+    function stakerStrategySharesIncludingPendingSlashings(address staker, IStrategy strategy) external view returns (uint256) {
+        address operator = delegation.delegatedTo(staker);
+        uint64 scalingFactor = slasher.pendingShareScalingFactor(operator, strategy);
+        return SlashingAccountingUtils.normalize({
+            nonNormalizedShares: nonNormalizedStakerStrategyShares[staker][strategy],
+            scalingFactor: scalingFactor
+        });
+    }
+
```

Comment:
Awful naming. Suggestions:
* `normalizedStakerShares`
* `pendingNormalizedStakerShares`

Other suggestion is to just have these methods live in the slasher.